scripts/validate/check-disallowed-permissions

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2162,SC2046,SC2002
+set -euo pipefail
+
+#
+# GovCMS disallowed permissions check.
+#
+# This will lint exported configuration files to identify if
+# permissions will been given to users via exports.
+#
+
+GOVCMS_ROLE_PATTERN=${GOVCMS_ROLE_PATTERN:-user.role.*.yml}
+GOVCMS_FILE_LIST=${GOVCMS_FILE_LIST:-}
+
+echo "GovCMS Validate :: Disallowed permissions"
+
+# We will need to export this during automated testing.
+if [ -z "${GOVCMS_FILE_LIST}" ]; then
```

Comment:
I've added this so that we can specify the file list in an export in the automated tests so that we can verify the logic below correctly.